go.mod

```diff
@@ -54,6 +54,9 @@ require (
 
 replace (
 	github.com/coreos/bbolt => go.etcd.io/bbolt v1.3.4
+	github.com/dubbogo/gost => github.com/wenxuwan/gost v1.9.4-0.20210309022905-7993b30b1a7d
```

Comment:
is it right replacment?

Reply:
No, just for test, when this pr https://github.com/dubbogo/gost/pull/47 merged, will replace it.